docs/proposals/PROPOSAL_C.md

```diff
@@ -321,3 +267,66 @@ Gives response:
+  - Yes: In this proposal, the existing image manifest layout does not change and can still be filtered by annotations.
+1. As a user, I want to fetch the most up-to-date artifact, collection of artifacts, or application.
+  - Partial: If Node manifests are used to combine artifacts of a certain type, and a client follows a pattern where the most up-to-date artifact is appended to the beginning of the list, then the most up-to-date artifact is the one at the beginning of the list. However, there isn't any strong protection against not following these rules. Clients will have to rely on some indicator in the description that points to the "latest" artifact. Another way to implement this is to have a Node reference pointing to an older Node reference. Then the first Node manifest represents the most recent artifact. The drawback of this mechanism is long Node chains which registries will have to garbage collect.
+1. As an artifact producer, I want to reduce the number of tags that reference an artifact.
+  - Yes: Node manifests pointing to a list of artifacts removes the requirement to tag the artifacts themselves if needed.
+
+#### Backwards Compatibility
+
+1. As a user, I want to be sure that existing container runtimes are not affected by any other type of registry artifact.
+  - Yes: Existing manifests such as container or index manifests are not affected.
+1. As a user, I want to move container images to and from registries that do not support reference types.
+  - Partial: Registries that do not support node manifests require the client to create tags for each of the embedded artifacts before pushing them. All the information in description and reference will be lost. Registries that support node manifests have no effect on artifacts that require tagging.
+1. As an artifact producer, I want to tag artifacts that I can pull by said tag, even if they contain references to other artifacts.
+  - Yes: Node manifests will not affect tags on other artifacts.
+1. As an artifact producer, I want be sure that pushing an artifact to a repository will not affect a copy of the same artifact previously created and referenced by a manifest list existing in another repository on the same registry.
+  - Yes: Node manifests do not modify existing artifacts nor do they create duplicates of the same artifact, as is the case with existing registries.
+1. As a tool writer, I want to identify whether a registry supports reference types or not.
+  - Yes: use `GET /v3/`. Catalog listing is not prescribed in this proposal.
+1. As a tool writer, I would like the option to perform a server side blob mount when copying a large artifact between repositories.
+  - Yes: If a large blob is pushed to the registry and then referenced by a Node manifest, a server side mount should be possible.
+1. As a tool writer, I want to be able to include reference types within the Image Layout filesystem format.
+  - Yes: The existing image layout will be unaffected.
```

Comment:
With client tooling that copies between registries and the layout spec, I'm not sure how I'd differentiate `/v2` and `/v3` in layout spec content.